content/en/docs/v3.5/integrations.md

```diff
@@ -32,8 +32,8 @@ The sections below list etcd client libraries by language.
 
 ### Go
 
-- [etcd/client/v3](https://github.com/etcd-io/etcd/tree/master/client/v3) - the officially maintained Go client for v3
-- [etcd/client/v2](https://github.com/etcd-io/etcd/tree/master/client/v2) - the officially maintained Go client for v2
+- [etcd/client/v3](https://github.com/etcd-io/etcd/tree/release-3.5/client/v3) - the officially maintained Go client for v3
```

Comment:
The 3.4 doc links to the `main/client/v3`, but here links to `release-3.5/client/v3`. Let's keep them consistent.

Reply:
Updated.